src/richie/apps/courses/api.py

```diff
@@ -182,7 +183,7 @@ def course_runs_sync(request, version):
         )
 
     # Look for the course targeted by the resource link
-    course_number = lms.extract_course_number(request.data)
+    course_number = normalize_code(lms.extract_course_number(request.data))
```

Comment:
Maybe write a migration to normalize existing codes ?
Because if some codes are not well formatted, `course_run_sync` will not found the course.

Reply:
Good point! I thought that in openedx we only use digits but you are right that technically, we should do this data migration. 
I just added the migration file to this PR.